.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2.1
+
+jobs:
+  linting:
+    machine:
+      image: ubuntu-2004:202107-02
+    steps:
+      - checkout
+      - run: docker-compose run composer install
+      - run: docker-compose run php70 ./vendor/bin/phpcs ./src
```

Comment:
Do you have a pointer to how this would work? I don't see any way to have

1. git
2. composer
3. php 5.5

this way without building a custom image. The documentation really gives me nothing.

At the end of the day I feel like I've spent more time trying to get this to work than we'd ever spend in builds on this library in the future.